### PR TITLE
Fix existing e-mail bug

### PR DESCRIPTION
**Pull request by ThomasOliverKimble**
Removes bug if user signs in with existing e-mail.
Can be tested in `testing.webots.cloud`

### CODE REVIEW

**Review by BenjaminDeleze — DISMISSED** [review]
Thank you

**Review by BenjaminDeleze — APPROVED** [review]

File: httpdocs/ajax/user/uniqueness.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  header('Content-Type: application/json');
+  $json = file_get_contents('php://input');
+  $data = json_decode($json);
+  require '../../../php/database.php';
+  $mysqli =  new mysqli($database_host, $database_username, $database_password, $database_name);
+  if (!$mysqli)
+    error("Can't connect to MySQL database: $mysqli->error");
+  $mysqli->set_charset('utf8');
+  $email = $mysqli->escape_string($data->{'email'});
+  $query = "SELECT * FROM user WHERE email like \"$email\"";
```
Comment (omichel) [inline httpdocs/ajax/user/uniqueness.php:14]: We use to use uppercase to distinguish SQL keywords, so "like" should be "LIKE".

File: httpdocs/ajax/user/uniqueness.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  $result = $mysqli->query($query) or error($mysqli->error);
+  $notunique = $result->fetch_assoc();
+  if ($notunique)
+    error("An account has already been created with this e-mail.");
+  else {
+    $status = 'OK';
+    die("{\"status\": \"$status\"}");
+  }
```
Comment (omichel) [inline httpdocs/ajax/user/uniqueness.php:22]: It would be simpler to have
```php
  else
    die('{"status":"OK"}');
```

Comment (ThomasOliverKimble) [reply httpdocs/ajax/user/uniqueness.php:22]: Will fix in next PR.

File: httpdocs/ajax/user/uniqueness.php (hunk 1 above)
Comment (ThomasOliverKimble) [reply httpdocs/ajax/user/uniqueness.php:14]: Will fix in next PR.

**Pull request merged**